src/signals/settings/users/Detail/components/UserForm/index.js

```diff
@@ -146,7 +212,7 @@ UserForm.propTypes = {
     }),
   }),
   onCancel: PropTypes.func,
-  onSubmitForm: PropTypes.func,
+  onSubmit: PropTypes.func,
```

Comment:
This function receives an object with a specific signature. Can you add a docblock that describes that param? See, for instance, the `AutoSuggest` component.

Reply:
Done

*cough* Typescript *cough*